CollAction/Controllers/ProjectsController.cs

```diff
@@ -415,7 +418,7 @@ public ViewResult StartInfo()
                     "Hi!<br><br>" +
                     "Thank you for participating in a CollAction project!<br><br>" +
                     "In crowdacting, we only act collectively when we meet the target before the deadline, so please feel very welcome to share this project on social media through the social media buttons on the project page!<br><br>" +
-                    "We�ll keep you updated on the project. Also feel free to Like us on <a href=\"https://www.facebook.com/collaction.org/\">Facebook</a> to stay up to date on everything CollAction!<br><br>" +
+                    "We�ll keep you updated on the project. Also feel free to Like us on <a href=\"https://www.facebook.com/collaction.org/\">Facebook</a> to stay up to date on everything CollAction!<br><br>" +
```

Comment:
I'm getting invalid unicode characters here in this change? Are you using a mac? We had issues like that before. Undo?